application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -96,15 +96,18 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.getFirst();
+        final int TITLE_MAX_LENGTH = 50;
```

Comment:
This constant seems pretty random and I am sure there is a JDA constant already, that you should use instead.

Reply:
it should be removed since its already defined at
https://github.com/Together-Java/TJ-Bot/blob/0b908d82c8672a9dcbb196525b6b60dcedfdebd1/application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java#L60

Reply:
I will remove the mentioned line. I didn't check the code thoroughly.